### DISCUSSION TITLE
Command to limit maximum number of decimal places to display varible or memory based values.

### DISCUSSION THREAD

**Discussion by jpier123** [arendst/Tasmota#19384] [General]
I'm looking for some advice about possible command option to limit maximum number of decimal places to display when talking about varible or memory based values. Similar to already supported VoltRes x; AmpRes x; TempRes x; WattRes x commands

By default, the dispaly will return three decimal places, which limits layout options when talking about 20x4 LCDs.

Rule example: 
on tele-T7#Temperature do Backlog var1 %value%;DisplayText [ds1f1l1c1]%var1%;DisplayText [s1f1l3c1]%mem1%;DisplayText [s1f1l3c7]%mem2% endon
on tele-T5#Temperature do Backlog var2 %value%;DisplayText [s1f1l1c7]%var2% endon
on tele-Counter#C2 do Backlog DisplayText [s1f1l1c13]Q%mem8%m3;DisplayText [s1f1l2c13]P%mem4%kW;DisplayText [s1f1l3c13]C%mem5% endon
on tele-T8#Temperature do Backlog var3 %value%;DisplayText [s1f1l2c1]%var3% endon
on tele-T6#Temperature do Backlog var4 %value%;DisplayText [s1f1l2c7]%var4% endon
on tele-Counter#C2 do Backlog mem1=(%var1%-%var3%);mem2=(%var2%-%var4%)  endon

Thank you in advance.

JP

**Comment by sfromis** [comment]
`CalcRes`

**Reply by jpier123** [reply]
I must just blind. Thanks a lot.
